src/dinterpret.d

```diff
@@ -4143,7 +4144,8 @@ public:
                 SliceExp oldse = cast(SliceExp)aggregate;
                 if (oldse.upr.toInteger() < upperbound + oldse.lwr.toInteger())
                 {
-                    e.error("slice [%d..%d] exceeds array bounds [0..%lld]", lowerbound, upperbound, oldse.upr.toInteger() - oldse.lwr.toInteger());
+                    e.error("slice [%llu..%llu] exceeds array bounds [0..%llu]",
+                        cast(ulong)lowerbound, cast(ulong)upperbound, oldse.upr.toInteger() - oldse.lwr.toInteger());
```

Comment:
That would defeat the whole purpose. There is no format for the lowerbound type, but there is for ulong.